sigmf/sigmffile.py

```diff
@@ -891,13 +923,13 @@ def get_dataset_filename_from_metadata(meta_fn, metadata=None):
 
 
 def fromarchive(archive_path, dir=None):
```

Comment:
This could also be easily modified to return _either_ a single SigMFFile or an iterable of them

Reply:
It would be nice for all of these to work:

```python
file_zero = some_sigmf.fromarchive(...)  # Single file case returns a single obj
file_one, file_two = some_sigmf.fromarchive(...)  # Multiple recording case can be unpacked
list_of_files = some_sigmf.fromarchive(...)  # Or just grab whole list of objects
```

Reply:
Fixed in d234ddf